modules/services/sxhkd.nix

```diff
@@ -73,6 +73,7 @@ in
     ];
 
     xsession.initExtra = ''
+      systemctl --user stop sxhkd.scope
```

Comment:
```suggestion
      systemctl --user stop sxhkd.scope || true
```

If an error occurs, is the script evaluation aborted?

Reply:
No, just throw an error to journal. I added check if scope is active.